TFT/src/User/Menu/PrintingMenu.c

```diff
@@ -187,12 +187,10 @@ void menuBeforePrinting(void)
   // initialize things before the print starts
   progDisplayType = infoSettings.prog_disp_type;
   layerDisplayType = infoSettings.layer_disp_type * 2;
-  setLayerNumber(0);
```

Comment:
This was a part of a bugfix. Why do you removed it?

Reply:
because "layer" and "total time" were moved to "infoPrinting" data struct that is initialized to 0 in printStart() function